cmd/keeper/cmd/keeper.go

```diff
@@ -641,7 +640,7 @@ func (p *PostgresKeeper) GetPGState(pctx context.Context) (*cluster.PostgresStat
 
 	initialized, err := p.pgm.IsInitialized()
 	if err != nil {
-		return nil, err
+		return pgState, err
```

Comment:
We could just do as the rest of the function, return the current pgstate (that will have healthy set to false) without returning an error. For the same reason we should change the signature of the function removing the error return since it will never return an error:

```
func (p *PostgresKeeper) GetPGState(pctx context.Context) *cluster.PostgresState
```

Reply:
OK, done below. But I'm confused that the function `GetPGState` is called in [stage of node resync](https://github.com/sorintlab/stolon/blob/master/cmd/keeper/cmd/keeper.go#L1279) and returned error interrupts the resync process obviously. How to deal with this issue? For now, I doesn't interrupt the resync process anyway.

Reply:
@sgotti  I have reverted the last commit and leave the first version